src/collective/elastic/plone/restapi/kitsearch/post.py

```diff
@@ -46,27 +47,14 @@ def search(self, data):
 
         query_body = data.get("elasticsearch_payload", {})
 
-        es_kwargs = dict(
+        search_kwargs = dict(
             index=INDEX_NAME,
             body=query_body,
         )
         if not query_body.get("size", None):
-            es_kwargs["size"] = 10
-        es = get_client()
-        try:
-            result = es.search(**es_kwargs)
-        except RequestError as e:
-            self.request.response.setStatus(500)
-            return dict(error=dict(message=e.message))
-        except TransportError as e:
-            self.request.response.setStatus(503)
-            return dict(error=dict(message=e.message))
-        except NotFoundError as e:
-            self.request.response.setStatus(404)
-            return dict(error=dict(message=e.body["error"]["reason"]))
-        except Exception as e:
-            self.request.response.setStatus(500)
-            return dict(error=dict(message=e.message))
+            search_kwargs["size"] = 10
```

Comment:
Removing the error handling here is ... why?

Reply:
Errors are now catched as is:
![grafik](https://github.com/collective/collective.elastic.plone/assets/1144737/6ff58118-0f09-4b5b-96b7-548005090995)

![grafik](https://github.com/collective/collective.elastic.plone/assets/1144737/fe6be01a-513b-44c1-9ba5-d637dc7ee03c)